### ISSUE TITLE
Screen interaction doesn't work

### ISSUE THREAD

**Issue by slav** [btimofeev/instead-launcher-android#16]
On Pixel 3 XL and Android 11. When launching any of the games, unable to interact with the screen. Pressing on the links in the game does nothing. Tried to enable cursor, it always stays in the top left corner.

The only thing I can do is open keyboard. Typing there works. However, when close keyboard the screen doesn't move back down it stays at the as if when it was pushed to the middle of the screen and space occupied by the keyboard remains black.

Not sure if these 2 issues are related.

Any advise on how to diagnose and potentially fix the issue?

**Comment by btimofeev** [comment]
Hello.  One user had a similar problem, it helped to minimize the game to the background and then go back to it.  I do not yet understand the reason for this behavior and how to solve it.

**Comment by timas130** [comment]
Xiaomi Redmi 5 Plus, Android 11. I have the same problem: when trying to click on anything, nothing happens. In the menu I can't click on anything either.

Seems to be just an Android 11 problem?

**Comment by btimofeev** [comment]
I have Android 11 on three phones and it works fine. This is a different problem. But until I have a phone with such a problem, I most likely cannot fix it. Minimizing the app doesn't help?

**Comment by timas130** [comment]
> Minimizing the app doesn't help?
No, I tried different games, minimizing, etc. Nothing, I can only click
on the keyboard and type without any problems.
